source/blog/2017-03-30-minishift-intro.html.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Introducing Minishift - Run OpenShift locally
+date: 2017-03-30 05:45 UTC
+author: budhram
+published: true
+comments: true
+tags: minishift, atomic-developer-bundle, openshift-local
+---
+
+It has been quite a few months since our [last blog post](http://www.projectatomic.io/blog/2016/08/vagrant-service-manager-1-3-0-release/) on [vagrant-service-manager](https://github.com/projectatomic/vagrant-service-manager) as we have been looking around the alternative of our [Atomic Developer Bundle (ADB)](https://github.com/projectatomic/adb-atomic-developer-bundle). It would be a separate blog post to address this reason.
+
+With this blog post, we are introducing [Minishift](https://github.com/minishift/minishift), an alternative to ADB, which is a tool that helps you run OpenShift locally by running a single-node OpenShift cluster inside a VM. You can try out OpenShift or develop with it, day-to-day, on your local host.
+
+Minishift is actually a single binary which uses [libmachine](https://github.com/docker/machine/tree/master/libmachine) to provisions VM and [OpenShift Origin](https://github.com/openshift/origin) for running the cluster.
+
+It's a platform (OS) agnostic CLI tool and supports multiple hypervisors like VirtualBox, Xhyve (Mac OS), VMWare Fusion (Mac OS) and HyperV (Windows). Detailed instructions about it can be found in the [prerequisites section](https://github.com/Minishift/minishift#prerequisites) of [Minishift's README](https://github.com/minishift/minishift#minishift).
```

Comment:
> add comma after agnostic.

`It's a platform (OS) agnostic CLI tool` is a single phrase.

> Why is KVM(fedora) not mentioned ?

This is generalized introduction of getting start. User can find more about OS specific in documentation. Hence didn't mentioned any OS.